NUITizenGallery/Examples/ScrollViewTest/ScrollViewTest7Page.xaml

```diff
@@ -28,19 +28,17 @@
                       Text="Random vertical scroll"
                       Size2D="{Binding Source={x:Static Window.Instance}, Path=Size }"
                       HeightSpecification="100"/>
-			<View x:Name="lblView">
-				<ScrollableBase x:Name="Scroller"
-                                  ScrollingDirection="Vertical"
-                                  HideScrollbar="false"
-                                  WidthSpecification="{Static LayoutParamPolicies.MatchParent}"
-                                  HeightSpecification="{Static LayoutParamPolicies.MatchParent}">
+            <ScrollableBase x:Name="Scroller"

```

Comment:
LGTM but alignment need to check once again.

Reply:
There were tab and spaces in the original file.
So I used spaces only for this patch.
If you see with visual studio code, then it would be fine.